src/ducks/Signals/signalFormManager/signalCrudForm/formParts/TriggerFormAssetWallet.js

```diff
@@ -119,18 +148,42 @@ const enhance = compose(
   graphql(pickGQL('all'), {
     name: 'Projects',
     props: mapDataToProps,
+    skip: ({ byAddress }) => {
+      return !!byAddress
+    },
```

Comment:
`[D-1-1]` Could be written with less syntax noise: `skip: ({byAddress}) => byAddress`.